src/main/java/seedu/address/logic/commands/HelpCommand.java

```diff
@@ -9,6 +9,8 @@
 
     public static final String COMMAND_WORD = "help";
 
+    public static final String USER_TIP = "You can find more commands using: " + COMMAND_WORD;
```

Comment:
```suggestion
    public static final String USER_TIP = "To find more commands, type: " + COMMAND_WORD;
```
maybe can vary the language a bit like this so it doesn't sound too monotonous